roles/pulp_webserver/templates/nginx.conf.j2

```diff
@@ -123,7 +123,7 @@ http {
         listen {{ pulp_webserver_http_port }} default_server;
         listen [::]:{{ pulp_webserver_http_port }} default_server;
         server_name _;
-        return 301 https://$host$request_uri;
+        return 301 https://$hostname$request_uri;
```

Comment:
Why this change?

Also, this looks unrelated.

Reply:
same as above